doc/website/advanced/installation-for-developers.md

```diff
@@ -0,0 +1,31 @@
+## Use Sanitizer Scan
```

Comment:
just as food for thought, since our users are also developer, wouldn't it be less ambiguous if this file would be called `installation-for-contirbutors.md`?

Reply:
Changed.